runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/walker/Rs2Walker.java

```diff
@@ -309,6 +310,7 @@ private static WalkerState processWalk(WorldPoint target, int distance) {
             }
             if (Rs2Player.getWorldLocation().distanceTo(target) < distance) {
                 setTarget(null);
+                sleep(300);
```

Comment:
Stall a subsequent action. There can be cases where subsequent invoke done in a short time span can not register.